### PR TITLE
quick-start: add guide

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** unknown

## Changes

I introduce this quick start for newbies. This quick start should provide a quick introduction for `tldr` pages via examples and not on formal rules.

I am going to standardize (#8252) everything and write formal rules in style guide, that's why we need smth that allow new users quickly edit `tldr` pages.

### CODE REVIEW

**Comment by EmilySeville7cfg** [comment]
This PR is replaced with #8366.

**Review by sbrl — COMMENTED** [review]
This looks cool, but what about the style guide? We really want a single source of truth, so that when we want to update things we don't have to look in multiple places.

File: contributing-guides/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- Print the current directory, and resolve all symlinks (i.e. show the "physical" path):
+
+`pwd --physical`
+
+- Print the current logical directory:
+
+`pwd --logical`
+```
+
+Although `tldr` pages use Markdown syntax we have to note some moments:
```
Comment (mfrw) [inline contributing-guides/quick-start.md:28]: How about:
```suggestion
Although `tldr` pages use Markdown syntax, we should note some conventions that we follow:
```

**Comment by EmilySeville7cfg** [comment]
> This looks cool, but what about the style guide? We really want a single source of truth, so that when we want to update things we don't have to look in multiple places.

There are already two sources of truth: our CONTRIBUTING.md and the style guide. 😄 Although the first one just summarizes the most important things of the latter one. We need to decide whether to enhance our CONTRIBUTING.md #8366 or whether to leave it unchanged while introducing this quick start.

**Comment by EmilySeville7cfg** [comment]
As #8366 is merged I am not sure whether this quick start is needed. Please reopen this PR is you have another point of view.

**Pull request closed without merging**